samples/Basic_Delegates/app/application.cpp

```diff
@@ -53,8 +53,8 @@ class Task
 		taskTimer
```

Comment:
Don't modify this file in the current PR.

Reply:
Fixed